packages/components/src/checkbox/src/Checkbox.css

```diff
@@ -39,43 +45,50 @@
     bottom: -1px;
     display: block;
     position: absolute;
-    box-shadow: 0 0 0 var(--o-ui-focus-ring-thickness-md) var(--o-ui-focus-ring-color-alias-default);
-    border-radius: var(--o-ui-br-2);
+    box-shadow: 0 0 0 0.12rem var(--hop-neutral-surface), 0 0 0 0.219rem var(--hop-primary-border-focus);
+    border-radius: var(--_checkbox-border-radius);
     transition: var(--o-ui-focus-ring-transition);
 }
 
 .o-ui-checkbox-checkmark {
     opacity: 0;
-    position: absolute;
-    top: -1px;
-    left: -1px;
 }
 
 .o-ui-checkbox-checked .o-ui-checkbox-box .o-ui-checkbox-checkmark {
-    fill: white;
+    fill: var(--hop-neutral-icon-active);
     opacity: 1;
 }
 
+.o-ui-checkbox-checked.o-ui-checkbox-disabled .o-ui-checkbox-box .o-ui-checkbox-checkmark {
+    fill: var(--hop-neutral-icon-disabled);
+}
+
 /* BOX | INDETERMINATE */
+.o-ui-checkbox-indeterminate .o-ui-checkbox-box {
+    --_checkbox-border-color: var(--hop-neutral-border-active);
+    --_checkbox-background-color: var(--hop-neutral-surface-active);
+}
+
 .o-ui-checkbox-indeterminate .o-ui-checkbox-box:after {
     content: "";
     position: absolute;
     top: 50%;
     left: 50%;
-    width: var(--o-ui-sz-3);
-    height: var(--o-ui-sz-3);
-    transform: translate(-50%, -50%) scale(0.33333);
-    background-color: var(--o-ui-bg-alias-accent);
+    width: 0.75rem;
+    height: 0.09375rem;
+    transform: translate(-50%, -50%);
+    background-color: var(--hop-neutral-icon-active);
 }
 
 /* BOX | INDETERMINATE | STATE | INVALID */
-.o-ui-checkbox-invalid.o-ui-checkbox-indeterminate .o-ui-checkbox-box:after {
-    background-color: var(--o-ui-bg-alias-alert);
+.o-ui-checkbox-invalid.o-ui-checkbox-indeterminate .o-ui-checkbox-box {
```

Comment:
I'm confused as to what the hover and focus states for invalid should be... Right now, you put it so that when you hover, there's no red:
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/343f558d-8334-4fc9-b757-eb0c25146faa)

Reply:
for example, in my select, I keep the red border. Now I'm wondering if this is ok:
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/39a2e2d5-0a09-4e8e-8996-b3cf60232396)

Reply:
You should be able to see the focus state even if something is invalid, the classic blue focus would do.

Reply:
I was showing the hover state, the blue border/box-shadow thing is fine, that's not the issue. It's the fact that it's invalid and hover. I'm wondering if we show the red border like I did in select or not show any red like in the checkbox.

Reply:
To me it should stay red.

Reply:
ok good and I got confirmation from Francis Benoit too. It should stay red.